### PR TITLE
Update docstrings for GAP.Packages functions

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: src/packages.jl [hunk 1]
```diff
@@ -87,11 +86,10 @@ function install(spec::String; interactive::Bool = true, pkgdir::AbstractString
 end
 
 """
-    update(spec::String; quiet::Bool = false)
+    update(spec::String; interactive::Bool = true, pkgdir::AbstractString = DEFAULT_PKGDIR, quiet::Bool = false)
```
Comment (ThomasBreuer) [inline src/packages.jl:89]: Perhaps write `GAP.Packages.DEFAULT_PKGDIR`, since the variable is not exported?

File: src/packages.jl [hunk 2]
```diff
@@ -123,19 +121,19 @@ end
 # thus nec. to start Julia anew
 
 """
-    remove(spec::String; quiet::Bool = false)
+    remove(spec::String; interactive::Bool = true, pkgdir::AbstractString = DEFAULT_PKGDIR, quiet::Bool = false)
```
Comment (ThomasBreuer) [inline src/packages.jl:124]: Perhaps write `GAP.Packages.DEFAULT_PKGDIR`, since the variable is not exported?

**Review by ThomasBreuer — APPROVED** [review]
Yes, documenting all arguments is an improvement.
If `interactive` gets documented, should we say that it means to pass this value to the `PackageManager` function in question?
(The meaning is quite subtle, `InstallPackage` updates an installed package only if `interactive` is `true` and the user confirms that the package shall get updated.)

**Comment by fingolfin** [comment]
@ThomasBreuer updated, please have another look

**Comment by ThomasBreuer** [comment]
@fingolfin Very good, thanks.

**Pull request merged**